### PR TITLE
fix xdsm test to handle non-existence of pdflatex

### PR DESCRIPTION
**Pull request by swryan**

### CODE REVIEW

File: openmdao/devtools/xdsm_viewer/xdsm_writer.py [hunk 1]
```diff
@@ -518,6 +522,14 @@ def number_label(number, text, alignment):
 
     if writer_name == 'pyxdsm':  # pyXDSM
         x = XDSMWriter()
+
```
Comment (swryan) [inline openmdao/devtools/xdsm_viewer/xdsm_writer.py]: I am leaning against doing the following here... I think it's better handled in pyXDSM and adding this logic would put the burden on us to add some awkward tests... thoughts?

Comment (Kenneth-T-Moore) [reply openmdao/devtools/xdsm_viewer/xdsm_writer.py]: This does seem like pyXDSM's responsibility.

File: openmdao/devtools/xdsm_viewer/xdsm_writer.py [hunk 2]
```diff
@@ -20,6 +20,15 @@
 
 from __future__ import print_function
 
+try:
+    from pyxdsm.XDSM import XDSM
+except ImportError:
+    msg = ('The pyxdsm package should be installed. You can download the package '
+           'from https://github.com/mdolab/pyXDSM')
+    raise ImportError(msg)
```
Comment (naylor-b) [inline openmdao/devtools/xdsm_viewer/xdsm_writer.py]: any chance we could include the pip install command here?  Something like:

    msg = ('The pyxdsm package should be installed. You can download the package '
           'from https://github.com/mdolab/pyXDSM, or install using the pip command: '
           'pip install git+https://github.com/mdolab/pyXDSM.git')

File: openmdao/devtools/xdsm_viewer/xdsm_writer.py (hunk 1 above)
Comment (naylor-b) [reply openmdao/devtools/xdsm_viewer/xdsm_writer.py]: I also think it's better handled by pyXDSM.

File: openmdao/devtools/xdsm_viewer/xdsm_writer.py (hunk 2 above)
Comment (swryan) [reply openmdao/devtools/xdsm_viewer/xdsm_writer.py]: done

File: openmdao/devtools/xdsm_viewer/xdsm_writer.py (hunk 1 above)
Comment (swryan) [reply openmdao/devtools/xdsm_viewer/xdsm_writer.py]: removed

**Pull request merged**